packages/atomic/src/components/commerce/atomic-commerce-load-more-products/e2e/atomic-commerce-load-more-products.e2e.ts

```diff
@@ -0,0 +1,77 @@
+import {Page, Response} from '@playwright/test';
+import {test, expect} from './fixture';
+
+test.beforeEach(async ({page}) => {
+  await page.goto(
+    'http://localhost:4400/iframe.html?id=atomic-commerce-load-more-products--in-page&viewMode=story'
+  );
+});
+
+test('should be A11y compliant', async ({loadMore, makeAxeBuilder}) => {
+  await loadMore.hydrated.waitFor();
+  const accessibilityResults = await makeAxeBuilder().analyze();
+  expect(accessibilityResults.violations).toEqual([]);
+});
+
+test('should display a recap of the amount of results', async ({loadMore}) => {
+  await await expect(loadMore.summary()).toBeVisible();
+});
+
+test('should display a load more button', async ({loadMore}) => {
```

Comment:
I just noticed that the default label on the button right now is "Load more results"... we should probably change that to "Load more products"

Same thing for the load more summary (and for the search / product listing summary for that matter, although this is arguably out of scope).